compass/ocean/tests/global_ocean/mesh/arm10to60/arm10to60.cfg

```diff
@@ -0,0 +1,57 @@
+# Options related to the vertical grid
+[vertical_grid]
+
+# the type of vertical grid
+grid_type = tanh_dz
+
+# Number of vertical levels
+vert_levels = 64
```

Comment:
@milenaveneziani and @darincomeau, the vertical grid for this mesh in legacy COMPASS was the 64-level version.  If you plan to use the 60-level version instead, please let me know.  We would replace this whole config section with:
```ini
# Options related to the vertical grid
[vertical_grid]

# the type of vertical grid
grid_type = 60layerPHC
```

Reply:
@xylar : the old ARRM mesh had 80 levels, and I think we would like to keep it that way.